File: BOJ/sloved.ac/새싹/문자열/java/2754.java
/*
2754번
학점계산

어떤 사람의 C언어 성적이 주어졌을 때, 평점은 몇 점인지 출력하는 프로그램을 작성하시오.
https://www.acmicpc.net/problem/2754
 */
import java.io.*;

class Main{
  public static void main(String[] args) throws IOException{
    BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    char first = (char)br.read();

    float score = 0;

    if(first == 'F'){
      System.out.println(0.0);
    } else{
      char second = (char)br.read();
      
      if(first == 'A') score += 4;
      else if(first == 'B') score += 3;
      else if(first == 'C') score += 2;
      else if(first == 'D') score += 1;

      if(second == '+') score += 0.3;
      else if(second == '-') score -= 0.3;

      System.out.println(score);
    }
  }
}

File: BOJ/solved.ac/새싹/문자열/python/2754.py
"""
2754번
학점계산

어떤 사람의 C언어 성적이 주어졌을 때, 평점은 몇 점인지 출력하는 프로그램을 작성하시오.
https://www.acmicpc.net/problem/2754
"""
grade = input()
score = 0

if(grade[0] == "A"):
    score += 4.0
elif(grade[0] == "B"):
    score += 3.0
elif(grade[0] == "C"):
    score += 2.0
elif(grade[0] == "D"):
    score += 1.0
else:
    print(0.0)
    exit()

if(grade[1] == "+"):
    score += 0.3
elif(grade[1] == "-"):
    score -= 0.3

print(score)